components/badge/style.ts

```diff
@@ -1,37 +1,48 @@
 import { css } from "emotion";
-import { getColors, getFonts } from "../../shared/style";
+import { getColors, getFonts } from "../../shared/styles/core";
 
-const badgeAppearances = {
+const badgeTheme = {
   "default": css`
     background-color: ${getColors().greyLight};
     border-color: ${getColors().greyLight};
     color: #1b2029;
   `,
   "success": css`
-    background-color: #14c684;
-    border-color: #14c684;
-    color: #ffffff;
+    background-color: ${getColors().green};
```

Comment:
Why wouldn't we explode the colors, like this


```js
const { green, white, yellow } = getColors();
...
const theme = {
	"success": css`
		background-color: ${green}
	`
}
```

Reply:
Like that idea.

Reply:
hah I forgot I can use `destructuring`. Thanks.